pkg/provider/github/detect.go

```diff
@@ -41,6 +41,11 @@ func (v *Provider) Detect(req *http.Request, payload string, logger *zap.Sugared
 	_ = json.Unmarshal([]byte(payload), &eventInt)
 
 	switch gitEvent := eventInt.(type) {
+	case *github.CheckSuiteEvent:
```

Comment:
probably we want to add this event to bootstrap command (by default enabled) and in docs for github app (in the list of events)

Reply:
ah nm, it's for the right of the token scope, for the event we need check_suite, will addr that thanks

Reply:
i added it, took the opportunity to increase coverage to make sure our cov reporter works :p